core/src/main/java/edu/wpi/grip/core/sockets/InputSocket.java

```diff
@@ -114,7 +114,14 @@ public void onValueChanged() {
 
     @Override
     public boolean equals(Object o) {
-      return o instanceof InputSocket && decorated.equals(o);
+      if (o instanceof InputSocket) {
```

Comment:
I don't think the previous version had to do it either.  Did not notice until now.